app/frontend/styles/submissions.scss

```
@@ -64,4 +64,205 @@
+  z-index: 2;
+}
+
+.action-col[data-mode="tutor"] {
+  width: 200px;
+  min-width: 200px;
+  max-width: 200px;
+  right: 200px;
+  background-color: white;
+  z-index: 2;
+
+  padding: 0.5rem;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)